fission/function.go

```diff
@@ -41,6 +42,19 @@ func fileSize(filePath string) int64 {
 	return info.Size()
 }
 
+func getPackageFileName(srcPkgName string, deployPkgName string) (string, error) {
+	if len(srcPkgName) == 0 && len(deployPkgName) == 0 {
+		return "", errors.New("Need --code or --package to specify deployment package, or use --srcpkg to specify source package.")
+	} else if len(srcPkgName) > 0 && len(deployPkgName) > 0 {
+		return "", errors.New("Cannot specify deployment package (--code, --package) and source package (--srcpkg) at the same time.")
```

Comment:
Hmm, I think we should allow this. E.g. users might just want to save the source along with a build, to allow debugging for example.